### DISCUSSION TITLE
npm start for the first time

### DISCUSSION THREAD

**Discussion by 1HyperZ** [Koenkk/zigbee2mqtt#13095] [General]
hi I'm trying to start the zigbee2mqtt using  the command npm start for the first time and I get this error Error: Error while opening serialport 'Error: Error Resource temporarily unavailable Cannot lock port'. I checked the possible solutions on the website and it tells me to run this This error occurs when another program is already using (and thus locking) the adapter. You can find out which via the following command: ls -l /proc/[0-9]/fd/ |grep /dev/ttyACM0 (replace /dev/ttyACM0 with your adapter port. When i run this like they are saying i get nothing 
pi@raspberrypi:/opt/zigbee2mqtt $ sudo ls -l /proc/[0-9]/fd/ |grep /dev/ttyUSB0
pi@raspberrypi:/opt/zigbee2mqtt $
it just continues what should i do

**Comment by dave21w** [comment]
Have you edited the configuration.yaml file, by default its lists the serial port as /dev/ttyUSB0
